lib/token.js

```diff
@@ -46,7 +46,7 @@ exports.generate = function (payload, secret, options = {}) {
     }
 
     const header = Object.assign(baseHeader, options.header);
-    const value = `${Utils.b64stringify(header)}.${Utils.b64stringify(content, options.encoding || 'utf8')}`;
+    const value = `${Utils.b64stringify(header)}.${Utils.b64stringify(content)}`;
```

Comment:
This option is used here, and should be able to be removed now: https://github.com/hapijs/jwt/blob/835ac389516e77dd6d4f82e9c318a62375072f16/test/token.js#L107

Reply:
removed